tests/conftest.py

```diff
@@ -331,6 +331,30 @@ def spark_session(test_backends):
         raise ValueError("spark tests are requested, but pyspark is not installed")
 
 
+@pytest.mark.order(index=2)
+@pytest.fixture(scope="module")
+def spark_warehouse_session(test_backends, tmp_path_factory):
+    if "SparkDFDataset" not in test_backends:
+        pytest.skip("No spark backend selected.")
+
+    try:
+        import pyspark
+        from pyspark.sql import SparkSession
+    except ImportError:
+        raise ValueError("spark tests are requested, but pyspark is not installed")
+
+    spark_warehouse_path = str(tmp_path_factory.mktemp("spark-warehouse"))
+    spark = get_or_create_spark_application(
+        spark_config={
+            "spark.sql.catalogImplementation": "in-memory",
+            "spark.executor.memory": "450m",
+            "spark.sql.warehouse.dir": spark_warehouse_path,
+        }
+    )
+    yield spark
```

Comment:
@lccasagrande This particular fixture's implementation is unclear to me; I am not sure what is being done -- would be delighted to discuss over a virtual meeting (please reach to me in the Great Expectations Slack Community).  Thank you.

Reply:
I did this to use the in-memory metastore instead of using Hive and persisting the files